### PR TITLE
Awsome books - plainJs

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
- Added simple UI Structure with HTML
- Added remove and add buttons with JavaScript
- Added list of books with JS
- Checked errors with linters

### CODE REVIEW

File: JS/main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+function newBook(e) {
+  e.preventDefault();
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  myBooks.push(new Book(title.value, author.value));
+  updateLocalStorage();
+  title.value = '';
+  author.value = '';
+}
+
```
Comment (arslanbisharat) [inline JS/main.js:40]: - [x] It should have a function that removes a book from the collection.
For Example: 
```
const books = [];

const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] The App should display all books saved in the collection in the top part of the page.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: JS/main.js (hunk 1 above)
Comment (petudeveloper) [reply JS/main.js:40]: Thanks for the feedback. On line 45 we have a `destroyBook` function that removes a book from a collection, is that what you are asking for?

**Comment by petudeveloper** [comment]
@arslanbisharat Thanks for the feedback. On line 45 we have a `destroyBook `function that removes a book from a collection, is that what you are asking for?

**Review by Lameck1 — APPROVED** [review]
Hi, @Hamzaoutdoors & @petumazo

Concerns in the preview addressed 👍🏼. All expected features for this task working well as tested. Your project is now complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**